src/library/Navigation/styled.js

```diff
@@ -0,0 +1,169 @@
+/* @flow */
+import { createStyledComponent } from '../styles';
+import { createThemedComponent } from '../themes';
+import Button from '../Button';
+import { ALIGN, INTERNAL_TYPE } from './constants';
+import { navigationTheme, navItemTheme } from './themes';
+
+export const NavigationRoot = createStyledComponent(
+  'nav',
+  ({ align, prefix, theme: baseTheme, type }) => {
+    const theme = navigationTheme(baseTheme);
+
+    const aligns = {
+      [ALIGN.start]: 'flex-start',
+      [ALIGN.center]: 'center',
+      [ALIGN.end]: 'flex-end',
+      [ALIGN.justify]: undefined
+    };
+
+    return {
+      backgroundColor: theme[`${prefix}Nav_backgroundColor${type}`],
+      borderBottom: theme[`${prefix}Nav_border`],
+      display: 'flex',
+      justifyContent: aligns[align],
+      listStyle: 'none',
+      margin: 0,
+      // prettier-ignore
+      padding: `${theme[`${prefix}Nav_paddingVertical`]} ${theme[`${prefix}Nav_paddingHorizontal`]}`,
+
+      ...(type === INTERNAL_TYPE.tabs
+        ? {
+            paddingBottom: 0
+          }
+        : undefined),
+
+      '& > :not(:first-child)': {
+        marginLeft: theme[`${prefix}Nav_gutter`]
+      }
+    };
+  },
+  {
+    filterProps: ['prefix'],
```

Comment:
I had left it off of NavigationRoot on purpose, as it could be either "PrimaryNav" or "SecondaryNav" and I thought just "Navigation" might be more confusing than helpful. Thoughts?

I added it to NavItemButton, though.

I think your hunch re: "label" might be right...

Reply:
Personally, I would still add it as the className otherwise ends up as something like `css-1ez53tx`, and with it at least helps you find the right neighborhood, but I don't feel very strongly about it.